HRTool/src/main/webapp/app/controllers/LoginController.js

```diff
@@ -0,0 +1,82 @@
+app.controller('LoginController', ['$scope', function($scope) {
+
+	$scope.onSignIn = function(googleUser) {
+
+	  	var profile = googleUser.getBasicProfile();
+			
+		var id_token = googleUser.getAuthResponse().id_token;  
+		  
+	  	console.log('ID: ' + profile.getId());
+	  	console.log('Name: ' + profile.getName());
+	  	console.log('Image URL: ' + profile.getImageUrl());
+	  	console.log('Email: ' + profile.getEmail());
+	  	console.log('id_token: '+ id_token);
+	  	
+	  	var request;
+	  	
+	  	request = $.ajax({
+	        url: "/signin/login",
+	        type: "POST",
+	        data: {"idtoken":id_token},
+	  	
+	  	
+	        success: function(data, textStatus, xhr) {
```

Comment:
What exactly is this supposed to be? Are you actually parsing JSON data... to JavaScript Object Notation ?